app/models/validators/feedback_answer_format_validator.rb

```diff
@@ -5,7 +5,7 @@ def validate(record)
     errors = record.errors[:answer]
 
     if kind =~ FeedbackQuestion.send(:intrange_regex)
-      range = ($1.to_i)..($2.to_i)
+      range = (Regexp.last_match(1).to_i)..(Regexp.last_match(2).to_i)
```

Comment:
At least with a open-source project, this enhances the Ruby ideology of readability. Special variables can seem a bit mysterious. However, I agree the shortcuts are handy in some cases.